perllib/FixMyStreet/Roles/CobrandBulkyWaste.pm

```diff
@@ -552,10 +552,14 @@ sub bulky_reminders {
 sub _bulky_send_reminder_email {
     my ($self, $report, $h, $params) = @_;
 
-    return unless $report->user->email;
+    return unless $report->user->email || $report->get_extra_field_value('bulky_text_updates');
```

Comment:
This is inside a function called `_bulky_send_reminder_email` so shouldn't have anything to do with sending texts - can you factor this (and the below `call_hook` lines) into a new `_bulky_send_reminder_text` function which gets called independently?